draft-ietf-quic-manageability.md

```diff
@@ -1023,6 +1023,38 @@ not arrive at their destination. Admitting a few packets allows the QUIC
 endpoint to determine that the path accepts QUIC. Sudden drops afterwards will
 result in slow and costly timeouts before abandoning the connection.
 
+## Guiding Path MTU
+
+Some networks are required to be able to carry 1500-byte packets but become
+inefficient at this size, because such packets are fragmented and
+reassembled across a segment where the effective MTU is smaller (e.g. due to
+IPSEC overhead). They do this even when the encapsulated packet is not a
+candidate for fragmentation by routers (it's IPv6, or has DF bit set). In such
+networks this leads to a need to influence a majority of senders to use smaller
+packets, so that the limited reassembly capacity is not exceeded.
+
+If the majority use TCP this can be managed with MSS Clamping, but QUIC
+requires a different approach. Section 14 of {{QUIC-TRANSPORT}} advises senders
```

Comment:
/majority use/majority of traffic use/
... although I really do think it better to avoid /can/ and say /is/... 

I suggest:

/For TCP, MSS-clamping is currently used to change the sender's maximum TCP segment size, but QUIC.../

Reply:
Happy with the new suggestion, so I'll merge it. I think I should also cite RFC4459 here.

Reply:
https://github.com/chris-box/ops-drafts/commit/4dbf7d8137ab17dc7de54644c2df18e6c172d380